packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -48,6 +50,16 @@ export class JaegerExporter implements SpanExporter {
       tags: jaegerTypes.ThriftUtils.getThriftTags(tags),
     };
     this._sender.setProcess(this._process);
+
+    const flushInterval = config.flushInterval || 5000;
+    this._timer = setInterval(() => {
+      this._sender.flush((numSpans: number, err?: string) => {
```

Comment:
Could you please create a private function for flush, looks like this is getting used in two places.